state_formatters/dp_formatters.py

```diff
@@ -1074,6 +1074,14 @@ def dff_prompted_skill_formatter(dialog, skill_name=None):
     )
 
 
+def dff_template_prompted_skill_formatter(dialog):
```

Comment:
Shouldn't this be dff_knowledge_prompted_skill?

Reply:
dff_knowledge_prompted_skill uses prompted_skill_formatter. Checked with dev, template_prompted_skill is no longer in formatters, so I deleted it.